package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "github",
   "main": "./lib/index",
-  "version": "0.20.0",
+  "version": "0.19.1-6",
```

Comment:
you probably want to change the version back before you merge, yes?

Reply:
Oh yeah definitely. 👍 

I might leave it as-is for the moment in case I have to do another prerelease. But now I'll have this comment to remind me :grin: